test/index.js

```diff
@@ -258,6 +261,26 @@ describe('Generic', function () {
       assert(type(obj) === 'foo', 'type(obj) === "foo"');
     });
 
+    var extendedArraySupported = false;
+    var extendedArrayBody = [
+      'class ExtendedArray extends Array {',
+      '  get [Symbol.toStringTag]() {',
+      '    return "ExtendedArray";',
```

Comment:
I'm not sure this should be a getter, `[Symbol.toStringTag]` is meant to be a function